src/core/config/tmf.h

```diff
@@ -45,6 +45,17 @@
 #define OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_ENTRIES 10
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_TMF_MAX_SNOOP_ADDRESS_CACHE_ENTRIES
+ *
+ * The maximum number of EID-to-RLOC cache entries that can be used for "snoop optimization" where an entry is created
+ * by inspecting a received message.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_TMF_MAX_SNOOP_ADDRESS_CACHE_ENTRIES
+#define OPENTHREAD_CONFIG_TMF_MAX_SNOOP_ADDRESS_CACHE_ENTRIES 2
```

Comment:
Just spotted one more nit (can be ignored):
 ```suggestion
#define OPENTHREAD_CONFIG_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES 2
```

Reply:
If yes.. then also change the order to define `OPENTHREAD_CONFIG_TMF_SNOOP_CACHE_ENTRY_TIMEOUT`.

Reply:
Great suggestion @LuDuda. Thanks.